Sources/AsyncServer.swift

```diff
@@ -1,9 +1,9 @@
 public protocol AsyncServer {
-    func serve(responder: AsyncResponder, on host: String, at port: Int) throws
+    func serve(_ responder: AsyncResponder, on host: String, at port: Int) throws
```

Comment:
But also, just to be clear: we’re not «serving responder», right? So maybe something like:
 ```swift
func serve(with responder: AsyncResponder, on host: String, at port: Int) throws
```
Will be good